tests/cephfs/cephfs_basic_tests.py

```diff
@@ -97,6 +97,10 @@ def run(ceph_cluster, **kw):
                 results.append("TC %s passed" % tc1)
 
             log.info("Execution of testcase %s started" % tc2)
+            client.exec_command(
+                cmd="sudo touch %s%s/%s"
```

Comment:
It is suggested to use `f-strings` to support uniformity in the code hence, could you change the way the format is done.
```
client.exec_command(sudo=True, cmd=f"touch {client_info["mounting_dir"]}{dir1}/test.txt")
```